js/misc/paygManager.js

```diff
@@ -92,6 +93,7 @@ var PaygManager = new Lang.Class({
 
         this._enabled = false;
         this._expiryTime = 0;
+        this._lastTimeAdded = 0;
```

Comment:
You should add a getter to encapsulate this private variable. See `get rateLimitEndTime()`.